client/src/pages/blog/components/unique_vists.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import Chart from 'chart.js/auto';
+import {getMetrics} from '../../../app/services/analytics.service.js';

```

Comment:
@suryavanshi18 The `getMetrics()` is a service written in the backend. Ideally, we would need to request for the data from the backend. In this case, we need to write a `fetch()` call to get the required data, else this wouldn't work.